src/scenic/syntax/scenic.gram

```diff
@@ -468,6 +468,10 @@ class Parser(Parser):
 '''
 
 
+# rule for adding hard keywords
+scenic_hard_keyword: 'PropertyDefault'
```

Comment:
We want to disallow users from using `PropertyDefault`. `scenic_hard_keyword` is a dangling rule just for defining hard keywords.

Reply:
OK. We can revisit this once we hide the imports from `veneer` (either putting some prefix in front of them or making them attributes of a single import with an unlikely name).